src/components/ColorAndIconPicker/filterIcons.ts

```diff
@@ -0,0 +1,16 @@
+import { Icon } from './useIconsQuery'
+
+export function filterIcons(icons: Icon[], filterText: string) {
+    const lowerText = filterText.toLowerCase()
+    return icons.filter(({ key, description, keywords }) => {
+        const matchesKey = key.indexOf(lowerText) > -1
+        const matchesDescription =
+            description && description.toLowerCase().indexOf(lowerText) > -1
```

Comment:
Isn't using `includes` a bit more idiomatic here?

Reply:
I simplified the whole thing.